src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -496,7 +499,41 @@ TEST_CASE(
     "correctly")
 {
   PerfAnalyzerParameters params{};
-  TestRequestRateManager trrm{params};
+  bool is_sequence_model{false};
+  bool async{false};
+  bool streaming{false};
+
+  const auto& ParameterizeAsyncAndStreaming{[](bool& async, bool& streaming) {
```

Comment:
This lambda helps me programmatically generate nested subcases for automatically creating more combinations of testing conditions.